train.py

```diff
@@ -27,11 +28,14 @@ def main(args):
 
     envs, datasets = load_envs(args.datasets) # Load Minari datasets and corresponding Gym environments
 
+    # load the text dataset
+    text_dataset = load_dataset('wikitext', 'wikitext-103-raw-v1')
```

Comment:
let's make this a bit more configurable. Potentially follow pattern in args.datasets (where one can pass a list of strings with the names of the control datasets), only load names of datasets specified in args, load none by default

Reply:
done in latest commit including adding Task class